community/2024-10-09-community-meeting.mdx

```diff
@@ -37,7 +37,20 @@ Today's meeting is focused on community roadmapping for Q4.
 * Feature suggestion: Deprecate `wash app list` in favor of `wash app get` since they use the same API - generally useful to consolidate similar subcommands and reduce the number you need to remember.
 * Feature suggestion: Address ComponentSpecification race conditions. 
 
--
+### Community Update
+
+- [Hacktoberfest](https://hacktoberfest.com/) is here! We've [tagged a ton of issues](https://github.com/wasmCloud/wasmCloud/issues?q=sort:updated-desc+is:issue+is:open+label:hacktoberfest) in the wasmCloud repo so you can add your wasmCloud contributions to your Hacktoberfest activities. Take a look and get started!
+
+### Tune In
+
+- If you’re going to Salt Lake City for WasmCon and KubeCon + CloudNativeCon NA why not tune in to the Rancher Labs podcast where our Taylor (and Salt Lake City local) will share his tips on visiting #SLC, and what to expect from this year’s conference.
+
+LinkedIn: [https://okt.to/EF5K29](https://t.co/x3YE4i0FRZ)
+YouTube: [https://okt.to/ACa9pQ](https://t.co/fPSz00J4K4)
```

Comment:
Ah it looks like we have a short link that is a link to a twitter link (which is a link to the LinkedIn page) -- is that right? 

Do we want to *only* use the Shortlink?

Reply:
Thanks for the flag, I will change that

Reply:
Yes, we only want to use the proper link to the thing itself

Reply:
Resolving! You should be able to merge now!